packages/eas-cli/src/update/utils.ts

```diff
@@ -19,40 +21,46 @@ export type UpdateGroupDescription = FormatUpdateParameter & {
   runtimeVersion: string;
 };
 
+export type FormattedUpdateGroupDescription = {
+  message: string;
+  group: string;
+  platforms: string;
+  runtimeVersion: string;
+};
+
+export type FormattedUpdateGroupDescriptionWithBranch = FormattedUpdateGroupDescription & {
+  branch: string;
+};
+
 export const UPDATE_COLUMNS = [
   'Update message',
   'Update runtime version',
   'Update group ID',
   'Update platforms',
 ];
 
-export function getUpdateGroupsWithPlatforms<
-  UpdateFragment extends Pick<Update, 'platform' | 'group'>
->(updates: UpdateFragment[]): (UpdateFragment & { platforms: string })[] {
-  return Object.values(groupBy(updates, updates => updates.group)).map(updateGroup => ({
-    ...updateGroup[0],
-    platforms: formatPlatformForUpdateGroup(updateGroup),
-  }));
-}
+export const UPDATE_COLUMNS_WITH_BRANCH = ['Branch', ...UPDATE_COLUMNS];
```

Comment:
not sure if folks prefer a different pattern here. I think standardizing our column/table layout is worthwhile and went with what seemed like the simplest option of multiple types.